### PR TITLE
Update README.md to include authentication with Jira Cloud

### PR DESCRIPTION
**Pull request by Eddie-Hartman**
Added Jira Cloud authentication steps to README.

### CODE REVIEW

**Comment by Eddie-Hartman** [comment]
#3 Resolves

**Comment by rybak** [comment]
Thank you for you contribution!

File: README.md [hunk 1]
```diff
@@ -60,6 +60,15 @@ With `config.py` in place, launch jira-gource:
 When the script starts downloading the tickets, it will prompt the user
 for their JIRA password to authenticate with the JIRA server.
 
+**If you are using Jira Cloud:**
+
+- You will receive a 401 code if you try to just authenticate with a password. You can easily use an API token instead.
```
Comment (rybak) [inline README.md]: Please wrap the text at 80 characters to make it more readable in raw format and in terminals.

File: README.md [hunk 2]
```diff
@@ -60,6 +60,15 @@ With `config.py` in place, launch jira-gource:
...
+- Go to [https://id.atlassian.com/manage/api-tokens](https://id.atlassian.com/manage/api-tokens) and create a new API token.
+- Copy the token to a local file because you will not have access to it after creating it.
+- When prompted for a password, use the API token instead.
+- Reference to the removal can be found here: [https://developer.atlassian.com/cloud/jira/platform/deprecation-notice-basic-auth-and-cookie-based-auth/](https://developer.atlassian.com/cloud/jira/platform/deprecation-notice-basic-auth-and-cookie-based-auth/)
```
Comment (rybak) [inline README.md]: Let's reword this bit and just put description as link text. Maybe something like "You can read about deprecation of basic authentication in Atlassian blog post".

**Review by rybak — CHANGES_REQUESTED** [review]

**Comment by rybak** [comment]
> #3 Resolves

You probably meant `Resolves #2`*. Let's put it in the commit message, and it would also be a good idea to include a direct link to [ https://github.com/rybak/jira-gource/issues/2](https://github.com/rybak/jira-gource/issues/2), since jira-gource is also hosted on other Git mirrors.

\* Footnote: per [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords) it seems that the number should always follow a keyword.

**Comment by Eddie-Hartman** [comment]
Thanks for the feedback. I haven't contributed on GitHub in a while.

**Pull request merged**